spec/features/user_profile_spec.rb

```diff
@@ -5,6 +5,10 @@
   let(:user2) { FactoryBot.create :user }
 
   before do
+    user.social_accounts.create(social_network: "GitHub", user_name: "SomeGitUserName")
```

Comment:
Would it be possible to use factories for these data?

Reply:
I will have a look at it. But not sure if today.